docs/markdown/pipelines/exomeseq.md

```diff
@@ -1,38 +1,54 @@
-Patient1    XY  1   Sample2 Lane3   /path/to/Data/Sample2_L003_R1.fastq.gz  /path/to/Data/Sample2_L003_R2.fastq.gz
-Patient2    XY  0   Sample3 Lane1   /path/to/Data/Sample3_L001_R1.fastq.gz  /path/to/Data/Sample3_L001_R2.fastq.gz
-Patient2    XY  1   Sample4 Lane1   /path/to/Data/Sample4_L001_R1.fastq.gz  /path/to/Data/Sample4_L001_R2.fastq.gz
+We usually start our analyses at QBiC from paired-end fastq files. For these files, so you can take this `input.csv` as example:
+
+```csv
+patient,sex,status,sample,lane,fastq_1,fastq_2
+patient1,XX,0,normal_sample,lane_1,/path/to/Data/test_L001_1.fastq.gz,/path/to/Data/test_L001_2.fastq.gz
+patient1,XX,0,normal_sample,lane_2,/path/to/Data/test_L002_1.fastq.gz,/path/to/Data/test_L002_2.fastq.gz
+patient1,XX,0,normal_sample,lane_3,/path/to/Data/test_L003_1.fastq.gz,/path/to/Data/test_L003_2.fastq.gz
+patient1,XX,1,tumor_sample,lane_1,/path/to/Data/test2_L001_1.fastq.gz,/path/to/Data/test2_L001_2.fastq.gz
+patient1,XX,1,tumor_sample,lane_2,/path/to/Data/test2_L002_1.fastq.gz,/path/to/Data/test2_L002_2.fastq.gz
+patient1,XX,1,relapse_sample,lane_1,/path/to/Data/test3_L001_1.fastq.gz,/path/to/Data/test3_L001_2.fastq.gz
 ```
 
 Columns:
 
-- _Subject_: the first column contains the subject ID. Samples belonging to the same patient / animal, should have the same subject ID.
-- _Sex_: the subject sex. XX for women, XY for men.
-- Status: 0 for normal samples, 1 for tumor samples. This is important for somatic variant calling.
-- _Sample ID_: sample ID should be unique across all samples. However, samples that are spread over several lanes should have the same ID.
-- _Lane ID_: if a sample is spread over multiple lanes, just enter the files individually in different rows, with the same sample ID but different lane IDs. In this example, Sample 1 is spread over 2 lanes, and Sample 2 over 3 lanes. Sample 3 and 4 are not spread over several lanes.
+- _patient_: the first column contains the subject ID. Samples belonging to the same patient / animal, should have the same subject ID.
+- _sex_ (optional, default: NA): the subject sex. XX for women, XY for men. This column is required for copy-number calling, for all other analysis types it is _optional_ and has no effect.
+- _status_ (optional, default: 0): 0 for normal samples, 1 for tumor samples. This is important for somatic variant calling.
+- _sample_: sample ID must be unique across all samples. However, samples that are spread over several lanes must have the same ID to be merged correctly. Depending on the type of experiment, the `Q_TEST_SAMPLE` is a good candidate to be used here.
+- _lane_: if a sample is spread over multiple lanes, just enter the files individually in different rows, with the same sample ID but different lane IDs. In this example, the normal sample is spread over 3 lanes, and the first tumor sample over 2 lanes. The relapse sample is not spread over several lanes.
```

Comment:
```suggestion
```

Reply:
Same here...

Reply:
At least for the sample using the qbic thing seems like useful info? I don't know how often it is used by the biopms or they use their own sample ids...